### ISSUE TITLE
Specific versions not honored by modules

### ISSUE THREAD

**Issue by magicalyak** [nginxinc/ansible-role-nginx#239]
Specifying a version in the free form nginx_version is honored by the nginx-plus install (and I guess OSS) but adding njs, for example, will always then update to the latest on centos7.  I assume this is similar for others.

**To reproduce**
1. Deploy nginx-plus with version specified for -20
2. see logs show nginx-plus-20 installs but when nginx-plus-njs is installed it automatically updates to 21

**Expected behavior**
Needs to respect the version if it works for nginx-plus it should for module install

**Your environment:**
 - Version of the NGINX Role or specific commit
```
ansible 2.9.7
  config file = /etc/ansible/ansible.cfg
  configured module search path = [u'/home/ubuntu/.ansible/plugins/modules', u'/usr/share/ansible/plugins/modules']
  ansible python module location = /usr/lib/python2.7/dist-packages/ansible
  executable location = /usr/bin/ansible
  python version = 2.7.17 (default, Nov  7 2019, 10:07:09) [GCC 7.4.0]
```
 - Target deployment platform
Centos7

**Additional context**
Found internally, will suggest a PR soon.  Either yum lock worst case or refactor the installs for the modules.

**Comment by magicalyak** [comment]
This PR fixes the issue but you'll want to use a wildcard to make it work with modules (we can't predict the naming).  So if you want nginx-plus r20 for example, you need to add the variable (and I added an example and # note
nginx_version: "-20*"
that will cover both the nginx-plus and all modules.  Otherwise the modules will auto-upgrade the nginx-plus
the other way to resolve is breaking out the package manager but that's a ton of work and adding technical debt.

**Comment by alessfg** [comment]
Fixed in #240 😄
